components/tutorial/tutorial_tip/tutorial_tip.tsx

```diff
@@ -256,8 +281,24 @@ export default class TutorialTip extends React.PureComponent<Props, State> {
         return this.targetRef.current;
     }
 
-    private getLastStep(tutorialSteps: Record<string, number>) {
-        return Object.values(tutorialSteps).reduce((maxStep, candidateMaxStep) => {
+    private getLastStep(tutorialSteps: Record<string, number>, category: string) {
+        let steps = tutorialSteps;
+
+        // temporary fix for filtering tutorial_step category based on role
+        // however this could work if TutorialSteps are converted to an enum and tips for only admins added last to the enum
+        if (category === Preferences.TUTORIAL_STEP) {
+            steps = Object.keys(steps).filter((key) => {
+                if (!this.props.isAdmin && key === 'START_TRIAL') {
```

Comment:
maybe we can here just return this condition evaluation `this.props.isAdmin && key === 'START_TRIAL'`

Reply:
Removed this code